<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogDialog</class>
 <widget class="QDialog" name="CatalogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справочники</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtnBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>329</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="CatalogWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>721</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>188</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="SearchLayout">
    <item>
     <widget class="QLabel" name="SearchLabel">
      <property name="text">
       <string>Поиск:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="CatalogLabel">
      <property name="text">
       <string>Справочник:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="CatalogComboBox">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="NewRecordBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая запись</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveChangesBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Внести изменения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteRecordBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>CatalogDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>CatalogDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
